lib/WebpackOptionsDefaulter.js

```diff
@@ -73,6 +73,11 @@ class WebpackOptionsDefaulter extends OptionsDefaulter {
 			{
 				test: /\.wasm$/i,
 				type: "webassembly/experimental"
+			},
+			{
+				test: /\.wast$/i,
```

Comment:
> The recommended file extension for WebAssembly code in textual format is .wat.

https://webassembly.org/docs/text-format/

Reply:
Actually wast (WebAssembly script format) and wat (text format) are two different syntaxes but i agree it's better to switch to `.wat`.